src/components/UI/Checkbox.js

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const CheckboxWrapper = styled.div``;
+
+const CheckboxLabel = styled.span`
+  margin-left: 5px;
+`;
+
+/* eslint-disable-next-line no-empty-function */
+const Checkbox = ({ checked, label, onChange = () => {} }) => {
```

Comment:
A prop's default value is usually set by using the `defaultProps` static property of a component.

https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values